_posts/2016-01-14-javascript-code-coverage-with-instanbul-and-coveralls.markdown

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: "JavaScript Code Coverage With Instanbul and Coveralls"
+date: 2016-01-14T21:50:18-02:00
+keywords: test,
+author: rafaeleyng
+excerpt: >
+  A fast lane to set up code coverage in your project
```

Comment:
I meant to change only the excerpt, the post itself doesn't need to be changed (neither does the excerpt, in fact - it was only a suggestion). Anyway, good job!